src/views/YourStocks.jsx

```diff
@@ -29,7 +29,7 @@ function YourStocks() {
   }, [dispatch, user]);
 
   useEffect(() => {
-    axios.get(`/stock/bank/${user?.id}`)
+    axios.get(`/stock/bank/${user.id}/${league}`)
```

Comment:
move this function to a separate method and then call that method inside useEffect